.azure-pipelines/azure-pipelines-UpgrateVersion.yml

```diff
@@ -40,10 +47,10 @@ stages:
     CACHE_MODE: none
     VERSION_CONTROL_OPTIONS: 'SONIC_VERSION_CONTROL_COMPONENTS='
   jobs:
-  - template: azure-pipelines-build.yml
+  - template: .azure-pipelines/azure-pipelines-build.yml@buildimage
     parameters:
       jobFilters: ${{ parameters.jobFilters }}
-      buildOptions: '${{ variables.VERSION_CONTROL_OPTIONS }} SONIC_BUILD_JOBS=$(nproc) ENABLE_IMAGE_SIGNATURE=y'
+      buildOptions: '${{ variables.VERSION_CONTROL_OPTIONS }} DEFAULT_CONTAINER_REGISTRY=publicmirror.azurecr.io SONIC_BUILD_JOBS=$(nproc) ENABLE_IMAGE_SIGNATURE=y'
```

Comment:
The DEFAULT_CONTAINER_REGISTRY can be added as a common variable template, shared by different branches.